src/katsdpmodels/primary_beam.py

```diff
@@ -771,6 +785,10 @@ def _sample_grid_altaz_jones(
         out_shape = frequency.shape + m.shape + l.shape + (2, 2)
         if out is None:
             out = np.empty(out_shape, np.complex64)
+            # Pre-fault the memory. This seems to speed up matrix
+            # multiplications that write out to this memory. See
+            # https://github.com/numpy/numpy/issues/18669
```

Comment:
So I'm going to start a text document "Bruce's Performance Improvements" where I record these techniques!